docs/CustomUIA.md

```diff
@@ -0,0 +1,115 @@
+<!-- Copyright (c) Microsoft Corporation. All rights reserved.
+     Licensed under the MIT Licence. -->
+
+## Custom UI Automation properties
+Axe Windows supports the inspection of [custom UI Automation properties](https://docs.microsoft.com/en-gb/windows/win32/winauto/uiauto-custompropertieseventscontrolpatterns). The custom properties for which Axe Windows should register are described in a user-supplied text file that is applied to all inspected UI Automation elements.
+
+### Configuration file format
+Custom UI Automation configuration data is specified in a [JSON](https://en.wikipedia.org/wiki/JSON#Syntax) formatted text file. The file consists of one object with the following possible attributes:
+
+#### metadata
```

Comment:
Do we have any plan right now for having axe-windows use this information directly? If not, it might be better to suggest users represent this as a comment instead, to avoid user confusion if we ever need to add a more strongly specified version field/etc.

Reply:
I added this as strict JSON has no support for comments.

Reply:
As discussed, let's remove this block in favor of comments